analyze_data.py

```diff
@@ -9,6 +9,16 @@
 import sys
 
 
+# Folder where reports are stored
```

Comment:
Smart to attempt to create folders if they don't exist.  But instead lets try to move this down to the 
```
if __name__ == "__main__"
```
section and try to make if not exists the in_folder and out_folder passed in via the command line.